x/feerefunder/types/genesis.go

```diff
@@ -19,6 +21,34 @@ func DefaultGenesis() *GenesisState {
 // failure.
 func (gs GenesisState) Validate() error {
 	// this line is used by starport scaffolding # genesis/types/validate
+	for _, info := range gs.FeeInfos {
+		addr, err := sdk.AccAddressFromBech32(info.Payer)
+		if err != nil {
+			return fmt.Errorf("failed to parse the payer address %v: %v", info.Payer, err)
```

Comment:
Pls use the `%w` to wrap the errors instead of `%v`

Reply:
Done